src/api/firebase.js

```diff
@@ -221,3 +225,74 @@ export async function deleteItem(listPath, itemId) {
 		throw new Error(`Failed updating item: ${error.message}`);
 	}
 }
+
```

Comment:
global urgency object will be populated with details and later set urgency state within List component

Reply:
It seems like leveraging context would be a great approach if the `urgencyObject` needs to be accessible across different components. Could we create a context for this and remove it from this file?

Reply:
Agreed, it was in firebase.js as per acceptance criteria, but the logic for useUrgency is inside hooks.js!